packages/web-components/src/components/va-date-text-input/va-date-text-input.tsx

```diff
@@ -0,0 +1,256 @@
+import { Component, Event, EventEmitter, Host, Prop, h } from '@stencil/core';
+
+import { days, isFullDate, checkLeapYear } from '../../utils/date-utils';
+@Component({
+  tag: 'va-date-text-input',
+  styleUrl: 'va-date-text-input.css',
+  shadow: true,
+})
+export class VaDateTextInput {
+  /**
+   * Render marker indicating field is required.
+   */
+  @Prop() required: boolean;
+
+  /**
+   * Label for the field.
+   */
+  @Prop() label!: string;
+
+  /**
+   * Used to create unique name attributes for each input.
+   */
+  @Prop() name!: string;
+
+  /**
+   * The error message to render (if any)
+   * This prop should be leveraged to display any custom validations needed for this component
+   */
+  @Prop() error: string;
+
+  /**
+   * The aria-describedby attribute lists the ids of the elements that describe the object.
+   * It is used to establish a relationship between widgets or groups and the text that describes them
+   */
+  @Prop() ariaDescribedby: string;
```

Comment:
I think @bkjohnson is saying we can add `aria-describedby` to the host, so it doesn't need to be a prop?

Reply:
@Mottie yes I believe so this would be something you would implement instead of having us bake it in. If you are ok with doing that please let me know and I will remove.

Reply:
Yes, that sounds good.

Reply:
Prop has been removed